Indexer/Indexer.php

```diff
@@ -70,8 +70,14 @@ class Indexer
 
     private $em;
 
-    public function __construct()
+    /**
+     * @var LoaderInterface
+     */
+    private $loaderChain;
```

Comment:
It's a loader. It can be a `YamlLoader`, an `AnnotationLoader` or a `ChainLoader` which contains any of the other.